util/cteutil/storage_test.go

```diff
@@ -14,78 +14,70 @@
 package cteutil
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
Our imports style follows goimports style, check https://goinbigdata.com/goimports-vs-gofmt/ and I think I should write a section for dev guide.